docs/API.md

```diff
@@ -257,8 +284,9 @@ Receiving a payload:
 Transmission types available in SRT
 -----------------------------------
 
-Mode settings determine how the sender and receiver functions work.
-The main socket options (see below for full description) that control it are:
+Mode settings determine how the sender and receiver functions work. The main 
+[socket options](#https://github.com/Haivision/srt/blob/master/docs/API-SocketOptions.md) 
```

Comment:
Dash in the name causes likely a wrong link.

OTOH I personally prefer the `API-SocketOptions.md` filename, you may consider renaming it.

BTW. I think it will be better if the links to the document use a relative, not network-absolute path.